### PR TITLE
[MDB Ignore] TG AND SKYRAT MAP RESET 14/09/21

### PR DESCRIPTION
**Pull request by Gandalf2k15**
Content loss percentage at ~40%

:cl:
refactor: All maps have been reset to TG standard as of 14/09/21
/:cl:

### CODE REVIEW

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -1384,7 +1384,6 @@
 
 		//SKYRAT EDIT START - RESEARCH DESIGNS
 		"medigunspeed",
-		"nightstick_conversion"
```
Comment (GoldenAlpharex) [inline code/modules/research/techweb/all_nodes.dm:1387]: This doesn't seem related.

Comment (Gandalf2k15) [reply code/modules/research/techweb/all_nodes.dm:1387]: Check CI of the other PR's

Comment (GoldenAlpharex) [reply code/modules/research/techweb/all_nodes.dm:1387]: Which one?

Comment (Gandalf2k15) [reply code/modules/research/techweb/all_nodes.dm:1387]: ## WARNING: Invalid research design with ID nightstick_conversion detected in node Electric Weapons[electronic_weapons] removed. in code/controllers/subsystem/research.dm at line 186 src: Research usr: .
[23:37:41] Runtime in unsorted.dm,1075: Invalid techweb nodes detected
  proc name: stack trace (/datum/proc/stack_trace)
  src: Research (/datum/controller/subsystem/research)
  call stack:
  Research (/datum/controller/subsystem/research): stack trace("Invalid techweb nodes detected")
  Research (/datum/controller/subsystem/research): initialize all techweb nodes(0)
  Research (/datum/controller/subsystem/research): Initialize(850612)
  Master (/datum/controller/master): Initialize(10, 0, 1)
